### ISSUE TITLE
Stop Simulation when Signal exceeds given value

### ISSUE THREAD

**Issue by gmeisl-twt** [into-cps/intocps-ui#269]
It would be nice if the COE could stop the simulation when a user selected signal exceeds a specified value. 
In our case we want to simulate a car drving a given route, but do not know how long it takes to cover this route. Currently we just simulate for a long time, however having a built-in option to stop on reaching the target (i.e. when a signal we can provide reaches or exceeds the value of 1) would be advantageous.
Because we can start the simulation directly via the http-interface, the availabilty of this option in the App is not essential for us (but of course would also be nice).

**Comment by ldcouto** [comment]
@lausdahl is this something we can consider in the scope of INTO?

**Comment by lausdahl** [comment]
Yes. There were a similar feature request in DESTECS (Crescendo) when this project was running. May I suggest to rais a requirement for this submitted based on knowledge from a case study.

The question is how to realise it. Naturally this leads to the discussion about the scripting language, so you can use expressions to describe when a stop should happen. The easy solution is to add the name of a boolean value which will stop the simulation if true. Alternatively for reals a standard expression ` x < y < z` where `xyz` can be numbers or signal names.

**Comment by gmeisl-twt** [comment]
I think for many use cases your suggestion to give a range (where one can put values or signals) would do the job. 
If more complicated expressions would be required it would also be possible to do such calculations within an FMU, which then again generates a signal which is simple to evaluate.

**Comment by ldcouto** [comment]
@lausdahl will you guys pursue this or should I archive the enhancement?
